nemo_text_processing/text_normalization/en/data/measurements.tsv

```diff
@@ -53,6 +53,7 @@ atm	atmosphere
 db	decibel
 ps	peta second
 oz	ounce
+fl	florida
```

Comment:
could you remind the sentence you encountered this? [I can't find such measure unit ](https://www.merriam-webster.com/dictionary/fl)

Reply:
```
------------------------------
INPUT : The terms differed little from those of Breda, except that the Republic undertook to pay a war indemnity of 2,000,000 fl. within three years.
GT    : The terms differed little from those of Breda, except that the Republic undertook to pay a war indemnity of two million florida. within three years.
WFST  : ["The terms differed little from those of Breda, except that the Republic undertook to pay a war indemnity of two million fl. within three years."]

Reply:
ok will remove, since this is weird